authentication/LoginHandler.js

```diff
@@ -28,6 +28,19 @@ export function LoginHandler(props) {
         prompt: "none",
         sessionToken,
       });
+      const result = await fetch(authUrl, {
```

Comment:
I think this part isn't really needed, as the devs will receive the error in the browser once they open it.
the problem with the current code is that it adds another request, and assumes that a 400 http status always means invalid redirect URI but we don't know that for sure, so it's best to avoid the assumption.